README.md

```diff
@@ -298,12 +318,38 @@ Finally, another distinguishing feature of Marko is that it supports _asynchrono
 
 # Usage
 
+## Template Loading
+
+Marko provides a custom Node.js require extension that allows Marko templates to be required just like a standard JavaScript module. For example:
+
+```javascript
+// The following line installs the Node.js require extension
+// for `.marko` files. Once installed, `*.marko` files can be
+// required just like any other JavaScript modules.
+require('marko/node-require').install();
+
+// ...
+
+// Load a Marko template by requiring a .marko file:
+var template = require('./template.marko');
```

Comment:
I just pushed another round of commits that change compiled templates to export a loaded `Template` instance. In the previous version of marko, compiled templates exported a "create" function that could be used to create a loaded `Template` instance. Now that compiled templates export a loaded `Template` instance, the compiled template module can be required to get a loaded `Template` instance and this will work the same on the server and in the browser (as long as the `template.marko.js` file gets transported and used as the require). Backwards compatibility has been maintained so that if a template was compiled with a previous version of marko then it will continue to be loaded just fine (tests were added).